zulipterminal/config/themes.py

```diff
@@ -4,33 +4,33 @@
 ThemeSpec = List[Tuple[Optional[str], ...]]
 
 required_styles = {
-    None,
-    'selected',
-    'msg_selected',
-    'header',
-    'custom',
-    'name',
-    'unread',
-    'user_active',
-    'user_idle',
-    'user_offline',
-    'user_inactive',
-    'title',
-    'time',
-    'bar',
-    'popup_contrast',
-    'msg_emoji',
-    'reaction',
-    'msg_mention',
-    'msg_link',
-    'msg_quote',
-    'msg_code',
-    'msg_bold',
-    'footer',
-    'starred',
-    'popup_category',
-    'unread_count',
-    'filter_results',
+    None: '',
```

Comment:
The explanatory comment that you had here earlier was helpful. Could we add it back?

Reply:
Done, with more description above.